.github/workflows/create-tag.yml

```diff
@@ -25,9 +27,17 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
         with: 
-          ref: ${{ github.event.inputs.ref }} 
+          ref: ${{ inputs.ref }} 
+          repository: ${{ inputs.repo }}
           
       - name: Tag the release
         run: |
-         git tag -a v${{ env.release-version}} -m "Release db-syncer:: ${{ env.release-version}}"
-         git push origin --tags
\ No newline at end of file
+         git tag -a v${{ inputs.ref }} -m "Release db-syncer:: ${{ inputs.ref }}"
+         git push origin --tags
+
+  publish:
+    uses: ./.github/workflows/publish-image.yaml
```

Comment:
I think we need todo `needs: tag` to ensure the two jobs are executed sequentially.